cypress/integration/tests/checkout/checkout.js

```diff
@@ -0,0 +1,31 @@
+import Chance from 'Chance';
+import CheckoutPage from '../../../page-objects/checkoutPage';
+import products from '../../../fixtures/products.json';
+import ShopPage from '../../../page-objects/shopPage';
+
+const chance = new Chance();
+
+describe('Checkout', () => {
+  beforeEach(() => {
+    cy.wrap(ShopPage.pickRandomProduct(products)).as('randomProduct');
```

Comment:
nope
please declare variable in a higher scope and then initialize it inside of `before` each.
your version works, but the usage of `this` is kinda confusing

Do not listen to your Indian guy ~_^)

Reply:
This time I was referring to Cypress documentation :))
I was first trying to do as you suggest, but I didn't realize that I should declare my variable outside of before Each and that's why it wasn't working for me.